recipes/breakfast/arabic_omelette.md

```diff
@@ -0,0 +1,35 @@
+# Arabic Omelette
+
+![Difficulty Badge](https://img.shields.io/badge/Difficulty-25%25-green.svg)
+![Speed Badge](https://img.shields.io/badge/Speed-10min-brightgreen.svg)
+![Reception Badge](https://img.shields.io/badge/Reception-Very%20Positive-brightgreen.svg)
+![Cost Badge](https://img.shields.io/badge/Cost-Very%20Cheap-brightgreen.svg)
+![Vegetarian Friendly Badge](https://img.shields.io/badge/Vegetarian-True-brightgreen.svg)
+![Vegan Friendly Badge](https://img.shields.io/badge/Vegan-False-red.svg)
+
+ **First cooked by:**
+[@Karyum](https://github.com/karyum)
+
+ **Date:** Week 5 FACN2
+ **Author of Recipe:**
+#MrsSaliba
+
+### Ingredients
+- 2 small eggs
+- a handful of parsley
+- a handful of green/spring onions
+- 1 teaspoon zaatar ( or to taste )
+- salt and pepper
+
+### Preparation
+
+1. Chop the parsley and the green/spring onions into small pieces.
+2. Add everything into a small bowl and whisk it with a *fork*.
+3. Heat up a pan with a bit olive oil and wait for 1 min untill it heats up.
+4. Add all of the mix into the heated pan.
+5. From here it's self preference on how much you want the omelette to be cooked, I would say flip it when it gets golden-brown.
+
+Tips:
+- The more you stir it the fluffier the omelette is going to get.
+- These are the base ingredients you could always get creative and add new ingredients on top of the base.
+- Don't screw up the flip.
```

Comment:
lol, funny that YOU say so !! you never flipped it right :roll_eyes:

Reply:
that's such a lie and you know it mate